app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="row">
+  <div class="col-md-6">
+    <h2 class="pb-4"><%= t('.sign_up') %></h2>
+
+    <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+      <%= render "devise/shared/error_messages", resource: resource %>
```

Comment:
The's something wrong with the registration process on your app. 

![image](https://user-images.githubusercontent.com/38088818/204083188-54fcd164-5623-4ad5-98a2-f57f014a1e76.png)

- Although I am still able to log in after the error. It is still best to make sure that there are no errors on your application to avoid confusion from users.